integration-tests/utils.ts

```diff
@@ -145,16 +146,16 @@ const expectSearchResultsToFulfillExpectation = async (
     search: { url: string; params?: any },
     bundleEntryExpectation: jest.Expect,
 ) => {
-    console.log('Searching with params:', search);
-    await expect(
-        (async () => {
-            return (
-                await client.get(search.url, {
-                    params: search.params,
-                })
-            ).data;
-        })(),
-    ).resolves.toMatchObject({
+    console.log('GET Searching with params:', search);
+    const searchResult = (await client.get(search.url, { params: search.params })).data;
+    expect(searchResult).toMatchObject({
+        resourceType: 'Bundle',
+        entry: bundleEntryExpectation,
+    });
+
+    console.log('POST Searching with params:', search);
+    const postSearchResult = (await client.post(`${search.url}/_search`, qs.stringify(search.params))).data;
```

Comment:
Could we add a test case of both in query param and in the post body?

Reply:
Also do we need qs here to stringify? the params would JSON.stringify work?

Reply:
I tried JSON.stringify first, it didn't work unfortunately.

Reply:
Aren't you using `qs` to have `Content-Type=application/x-www-form-urlencoded`?

Reply:
JSON.stringify and qs.stringify do different things actually. If we take `{ x: 5, y: 6 }` as example, JSON.stringify would give `"{"x":5,"y":6}"`, and qs.stringify would give `"x=5&y=6"`.